charts/argo-cd/templates/argocd-repo-server/metrics-service.yaml

```diff
@@ -9,12 +9,7 @@ metadata:
   {{- end }}
 {{- end }}
   labels:
-    app.kubernetes.io/name: {{ template "argo-cd.repoServer.fullname" . }}-metrics
```

Comment:
I want to highlight this because this one was different from other `name` labels (used fullname template). This will change to default name pattern.